Software/lib/GNSE_BSP/GNSE_bsp_gpio.h

```diff
@@ -132,6 +132,22 @@ typedef enum
 #define LOAD_SWITCH2_GPIO_CLK_ENABLE() __HAL_RCC_GPIOC_CLK_ENABLE()
 #define LOAD_SWITCH2_GPIO_CLK_DISABLE() __HAL_RCC_GPIOC_CLK_DISABLE()
 
+/*!
+ * Load Switch delay time defines in ms
+ * There are two load switches: one for the flash IC, the other for the sensors and secure element
+ * All values are rounded up (and increased) to ensure functionality.
+ * Delay values (Load Switch and sensor have to be added together for the full delay time)
+ * Load Switch:                 typ. 120-200 us
+ * Flash IC:                    min. 800 us
+ * Secure Element:              min. 100 us
+ * Temperature/Humidity sensor: max. 240 us
+ * Accelerometer:               tested to 5 ms, but could vary in different temperatures (datasheet does not define power on time)
+ */
+#define FLASH_LOAD_SWITCH_DELAY_MS 5U
```

Comment:
```suggestion
#define LOAD_SWITCH_FLASH_DELAY_MS 5U
```
To preserve the naming and keep it similar to `LOAD_SWITCH_SESNORS` or `LOAD_SWITCH_FLASH`